backend/src/modules/lobby/draft/draftState.js

```diff
@@ -81,12 +82,22 @@ export const getDraftState = async ({
     activeDraftOrder = draftState.max_players - activeDraftOrder + 1;
   }
 
+  userOnTurn = activeDraftOrder === draftOrder.draft_order ? true : false;
+
+  if (userOnTurn) {
+    secondsToNextRound = (totalRounds + 1) * draftState.draft_round_limit;
+  }
+
   const timeOfNextRound = new Date(
     draftState.draft_start_at.getTime() +
       1000 * draftState.draft_time_offset +
       1000 * secondsToNextRound,
   );
   const timeLeft = (timeOfNextRound - Date.now()) / 1000;
+  let finished = false;
+  if (totalRounds >= draftState.max_players * 18) {
```

Comment:
Tohle cislo by mozna melo jit z DB, kde to bude nakonfigurovany na konkretni sport/udalost ne?

Reply:
@deny1994  ano určitě. chtěl jsem to fixnout jen zatím napevno ... je to TODO ... a taky by bylo lepší to čekovat na záčtku toho dotazu na draft, ale nevěděl jsem jak to v téhle draftState funkci vyřešit tak aby se to nemuselo složitě větvit a vracelo by to stejná data .. pokud myslíš že je to takhle ok a jen načítat číslo z DB tak to předělám

Reply:
přidán sloupec do db ke hře .. po dohodě s Petrem